src/walltime.h

```diff
@@ -8,15 +8,14 @@ typedef double WallTime;
 
 namespace walltime {
 WallTime Now();
-
-// GIVEN: walltime, generic format string (as understood by strftime),
-// a boolean flag specifying if the time is local or UTC (true=local).
-// RETURNS: the formatted string. ALSO RETURNS: the remaining number of
-// microseconds (never printed in the string since strftime does not understand
-// it)
-std::string Print(WallTime time, const char *format, bool local,
-                  int *remainder_us);
 }  // end namespace walltime
+
+std::string DateTimeString(bool local = false);
```

Comment:
At this point? No. If a reporter wants to report a GMT time (as opposed to local) then it or something similar must be exposed.